### ISSUE TITLE
There is a Cross Site Scripting(XSS) Vulnerability

### ISSUE THREAD

**Issue by huy4ng** [timeromantic/hotDataList#32]
I watched a odd title when I visited your website

**Comment by opop007** [comment]
![image](https://user-images.githubusercontent.com/22022699/64158814-be3cf800-ce6b-11e9-9a74-bf1ceab73f03.png)

**Comment by huy4ng** [comment]
Yes, It is .  I uploaded my pic failed and I have try many time. lol

**Comment by timeromantic** [comment]
Thank you for your message. The XSS on the web page has been repaired.

**Comment by NicoleG25** [comment]
> Thank you for your message. The XSS on the web page has been repaired.

@timeromantic could you specify where the fix was made?
Note that [CVE-2019-20377](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-20377) was assigned to this issue.
Thank you.
